relay/traffic/plugin-loader/loader.go

```diff
@@ -31,6 +40,24 @@ func LoadPlugins(
 	return trafficPlugins, nil
 }
 
+// pluginFactoryIsRegistered returns true if the provided plugin factory appears
+// in one of the groups of traffic plugins in registry.go. Checking this helps
+// ensure that newly-developed plugins get registered and are available for use
+// in production. (And not just, say, in unit tests.)
+func pluginFactoryIsRegistered(factory traffic.PluginFactory) bool {
```

Comment:
I figured that this sanity check would make it a little clearer to future developers what needs to be done to get a plugin registered with the relay. To make this work without a circular import, I had to move this stuff into a new `plugin-loader` package.